README.md

```diff
@@ -2,9 +2,130 @@
+
+```gradle
+dependencies {
+    implementation("com.navercorp.eventeria:spring-boot-eventeria:${version}")
+}
+```
+
+```java
+@Configuration
+public class MessageConfig {
+    @Bean
+    ChannelBindable channelBindable() {
+        return new ChannelBindable();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    ChannelBinder channelBinder(
+        SubscribableChannelBindingTargetFactory bindingTargetFactory,
+        ChannelBindable channelBindable
+    ) {
+        return new DefaultChannelBinder(bindingTargetFactory, channelBindable);
+    }
+}
+```
+
+## Produce `Message`s
+
+1. Register your spring-cloud-stream produce channel in application.yml or application.properties
+    ```yml
+    spring:
+      cloud:
+        stream:
+          bindings:
+            outbound-kafka-channel-command:
+              destination: command_topic
+              producer:
+                partition-count: 2
+    ```
+2. Register beans of outbound `MessageChannel`, `SpringMessagePublisher`, `IntegrationFlow`.
+    - [example code](https://github.com/naver/eventeria/blob/main/guide-projects/spring-boot-eventeria-guide/src/main/java/com/navercorp/eventeria/guide/boot/publisher/ProgrammaticBindingNotifyCommandPublisher.java)
+3. Inject `SpringMessagePublisher` bean and publish messages.
+
+```java
+@Service
+public class ExampleService {
+    ExampleService(
+        @Qualified("springMessagePublisherBean") SpringMessagePublisher springMessagePublisher
+    ) {
+        this.springMessagePublisher = springMessagePublisher;
+    }
+    
+    public void publish(Message message) {
+        springMessagePublisher.publish(message);
+    }
+}
+```
+
+> **NOTE:** Does not support Functional publishing currently.
+
+## Consume `Message`s
+
+1. First, register your spring-cloud-stream consume channel in application.yml or application.properties
+    ```yml
+    spring:
+      cloud:
+        stream:
+          bindings:
+            inbound-kafka-channel-command:
+              group: example_group
+              destination: command_topic
+              consumer:
+                concurrency: 1
+    ```
+2. Choose one of the Programmatic or Functional way to consume `Message`s.
+
+### Programmatic way
+
+- Register beans of inbound `SubscribableChannel`, `SpringMessagePublisher`, `IntegrationFlow`. `MessageRouter`
+   - [example code](https://github.com/naver/eventeria/blob/main/guide-projects/spring-boot-eventeria-guide/src/main/java/com/navercorp/eventeria/guide/boot/listener/ProgrammaticBindingEventListener.java)
+   - `MessageRouter`: route messages to your application.
+
+### Functional way
+
+1. Register additional function mappings in application.yml or application.properties.
+   ```yml
+   spring:
+      cloud:
+        function:
+          definition:
+            transformCloudEventToMessage|consumeImplementsEventeriaMessage;
+            transformCloudEventToMessage|consumeXXX;
```

Comment:
`transformCloudEventToMessage` is a function not provided by Eventeria, right?

Reply:
Yes. The guide exists in 'Functional way > 2'